timeline.py

```diff
@@ -0,0 +1,35 @@
+import fileinput
+
+
+class Timeline:
+
+    # TODO: Once the program is moved to .git, change the path to undo.timeline
+    def __init__(self, path='.git/undo.timeline'):
+        self._path = path
```

Comment:
Maybe we should talk and agree on if we should use or not use _ for class variables.